plugin-fake/cmd/main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-plugin"
+	"github.com/josephburnett/sk-plugin/pkg/skplug"
+	"github.com/josephburnett/sk-plugin/pkg/skplug/proto"
+)
+
+const (
+	pluginType = "fake-plugin"
+)
+
+type partition string
+type pod_name string
+
+var _ skplug.Plugin = &fakePluginServer{}
+
+type fakePluginServer struct {
+}
+
+func newPluginServer() *fakePluginServer {
+	return &fakePluginServer{}
+}
+
+type PartitionError struct {
+	error
+}
+
+func (pe *PartitionError) Error() string {
+	return "non-existent autoscaler partition error"
+}
+
+func NewPartitionError(s string) error {
+	return &PartitionError{}
+}
```

Comment:
What's the point of accepting a string and not using it?

Reply:
Yeah, looks weird. Forgot to remove it. Anyway, fixed it.